Mathlib/Algebra/BigOperators/Group/Finset/Basic.lean

```diff
@@ -162,6 +162,15 @@ theorem prod_filter_xor (p q : α → Prop) [DecidablePred p] [DecidablePred q]
   classical rw [← prod_union (disjoint_filter_and_not_filter _ _), ← filter_or]
   simp only [Xor']
 
+/-- In a salient monoid (whose only unit is `1`), a product is equal to `1` iff all factors are `1`.
```

Comment:
Mathlib usually refers to these as uniqueUnits in def/lemma names. Here I agree that's not sensible, but let's use that in the comment rather than the new term "salient", which seems very non-standard

Reply:
Salient monoid certainly isn't standard though!
Further, since there's already a way of talking about these in mathlib, it's preferable to use that rather than introduce a new one, which is hard to find anything about online. 
In your reference, the index makes no mention of "salient" nor is "salient" listed under the "cone" entry. Doing a search for "salient" in the textbook also returns no results. Is that reference the one you actually meant?

Reply:
Cox-Little-Schenck calls them pointed, but mathlib calls them salient: [ConvexCone.Salient](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/Convex/Cone/Basic.html#ConvexCone.Salient)

Reply:
Mathlib calls the cones salient, not the monoids.